alacritty/src/event.rs

```diff
@@ -51,11 +51,15 @@ pub struct DisplayUpdate {
     pub dimensions: Option<PhysicalSize<u32>>,
     pub message_buffer: Option<()>,
     pub font: Option<Font>,
+    pub cursor: Option<()>,
 }
 
 impl DisplayUpdate {
     fn is_empty(&self) -> bool {
-        self.dimensions.is_none() && self.font.is_none() && self.message_buffer.is_none()
+        self.dimensions.is_none()
+            && self.font.is_none()
+            && self.message_buffer.is_none()
+            && self.cursor.is_none()
```

Comment:
Quick question, do you know why we use `Option<()>` instead of `bool` for `message_buffer` and `cursor`?

I don't remember having a specific reason to do so and both are basically just single bit invariants.

Reply:
I have no idea, but I just followed the old code here for consistency. I feel like we can use `bool`.